diff remote[auth] local[auth]
--- remote[auth]
+++ local[auth]
@@ -47,13 +47,13 @@
 inactivity_timeout = "0s"
 
 [email]
-enable_signup = true
-double_confirm_changes = true
-enable_confirmations = true
-secure_password_change = true
-max_frequency = "1s"
-otp_length = 6
-otp_expiry = 3600
+enable_signup = false
+double_confirm_changes = false
+enable_confirmations = false
+secure_password_change = false
+max_frequency = "1m0s"
+otp_length = 8
+otp_expiry = 86400
 [email.template]
 [email.template.confirmation]
 content_path = ""
@@ -69,25 +69,25 @@
 content_path = ""
 [email.notification]
 [email.notification.email_changed]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.identity_linked]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.identity_unlinked]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.mfa_factor_enrolled]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.mfa_factor_unenrolled]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.password_changed]
-enabled = true
+enabled = false
 content_path = ""
 [email.notification.phone_changed]
-enabled = true
+enabled = false
 content_path = ""
 
 [sms]
