diff remote[auth] local[auth]
--- remote[auth]
+++ local[auth]
@@ -24,23 +24,23 @@
 
 [hook]
 [hook.mfa_verification_attempt]
-enabled = true
+enabled = false
 uri = "pg-functions://postgres/public/verifyMFA"
 secrets = ""
 [hook.custom_access_token]
-enabled = true
+enabled = false
 uri = ""
 secrets = ""
 [hook.send_sms]
-enabled = true
+enabled = false
 uri = "https://example.com"
 secrets = ""
 [hook.send_email]
-enabled = true
+enabled = false
 uri = ""
 secrets = ""
 [hook.before_user_created]
-enabled = true
+enabled = false
 uri = ""
 secrets = ""
 
