diff remote[auth] local[auth]
--- remote[auth]
+++ local[auth]
@@ -25,16 +25,16 @@
 [hook]
 
 [mfa]
-max_enrolled_factors = 10
+max_enrolled_factors = 0
 [mfa.totp]
 enroll_enabled = false
 verify_enabled = false
 [mfa.phone]
-enroll_enabled = false
-verify_enabled = false
-otp_length = 6
-template = "Your code is {{ .Code }}"
-max_frequency = "5s"
+enroll_enabled = true
+verify_enabled = true
+otp_length = 0
+template = ""
+max_frequency = "0s"
 [mfa.web_authn]
 enroll_enabled = false
 verify_enabled = false
