diff remote[auth] local[auth]
--- remote[auth]
+++ local[auth]
@@ -84,7 +84,7 @@
 
 [external]
 [external.apple]
-enabled = false
+enabled = true
 client_id = "test-client-1,test-client-2"
 secret = "hash:ce62bb9bcced294fd4afe668f8ab3b50a89cf433093c526fffa3d0e46bf55252"
 url = ""
@@ -91,9 +91,9 @@
 redirect_uri = ""
 skip_nonce_check = false
 [external.azure]
-enabled = false
-client_id = ""
-secret = ""
+enabled = true
+client_id = "test-client-1"
+secret = "hash:ce62bb9bcced294fd4afe668f8ab3b50a89cf433093c526fffa3d0e46bf55252"
 url = ""
 redirect_uri = ""
 skip_nonce_check = false
@@ -140,7 +140,7 @@
 redirect_uri = ""
 skip_nonce_check = false
 [external.google]
-enabled = true
+enabled = false
 client_id = "test-client-2"
 secret = ""
 url = ""
