diff remote[auth] local[auth]
--- remote[auth]
+++ local[auth]
@@ -11,15 +11,14 @@
 
 [hook]
 [hook.mfa_verification_attempt]
-enabled = true
+enabled = false
 uri = ""
 [hook.password_verification_attempt]
 enabled = false
 uri = ""
 [hook.custom_access_token]
-enabled = false
-uri = ""
-secrets = "hash:b613679a0814d9ec772f95d778c35fc5ff1697c493715653c6c712144292c5ad"
+enabled = true
+uri = ""
 [hook.send_sms]
 enabled = false
 uri = ""
